src/main/java/signup_and_login/UserPresenter.java

```diff
@@ -0,0 +1,24 @@
+package signup_and_login;
+
+import javax.swing.*;
+
+public class UserPresenter implements UserOutputBoundary {
+
+    /**
+     * Displays the JoinOrHostView passing in the specified user ID.
+     * @param userID the ID of the user
+     */
+    @Override
+    public void prepareJoinOrHostView(int userID) {
```

Comment:
Is there any reason for presenting userID instead of username to users? I think displaying username on the screen would be better for users since they only know their username

Reply:
The user will not actually see that pop-up window, that's just there to give visual feedback after a successful sign in while we wait on the implementation of JoinOrHostView. As for why we are passing in UserID instead of Username into JoinOrHostView please see Alec's review above.